test/iteration_1_test.rb

```diff
@@ -1,31 +1,26 @@
 require 'minitest/autorun'
 require 'minitest/pride'
-require 'pry'
 require_relative "../lib/district_repository"
-require_relative "../lib/enrollment_repository"
 
 class IterationOneTest < Minitest::Test
-  attr_reader :dr
-
-  def fixture_path
+  def kindergarten_data_fixture_path
     File.expand_path("fixtures/Kindergartners in full-day program.csv", __dir__)
   end
 
   def setup
```

Comment:
Moving our public class variable into an instance variable is a best case practice, like above.